src/PostgREST/Middleware.hs

```diff
@@ -64,7 +64,7 @@ runPgLocals conf claims role app req jsonDbS actualPgVersion = do
     roleSql = [setConfigLocal mempty ("role", toUtf8 role)]
     appSettingsSql = setConfigLocal mempty <$> (join bimap toUtf8 <$> configAppSettings conf)
     searchPathSql =
-      let schemas = T.intercalate ", " (iSchema req : configDbExtraSearchPath conf) in
+      let schemas = "\"" <> T.intercalate "\", \"" (iSchema req : configDbExtraSearchPath conf) <> "\"" in
```

Comment:
This a great fix @laurenceisla, really nice tests!

WDYT about having a separate function for this logic, it would be similar to `pgFmtIdent`

https://github.com/PostgREST/postgrest/blob/d87b622d64d993a240be9a28b7e6f515a489c161/src/PostgREST/Query/SqlFragment.hs#L152-L153

Reply:
> WDYT about having a separate function for this logic, it would be similar to pgFmtIdent

Thanks to this I noticed that double quotes `"` still weren't being accepted. I used the `pgFmtIdent` function to build the list now.